### PR TITLE
Add hyperlink for commands

### LINKED ISSUE #1324: Linking Docs & Similar commands
Hi team,

Currently, if we mention another command in one command's documentation, it is not hyperlinked, hence user has to look up and go to the page manually. We would like to have all such mentions linked to each other via links for improved user experience. The scope of the issue is all the commands. For example:

image
In BITFIELD_RO, we mention BITFIELD  but it is not a link. A link  would have improved the experience. There are many such mentions.

Acceptance Criteria:
- All such commands, except those not documented but mentioned, are linked to each other via relative URLs.
- Any command having another command which have same functionality, for example, HMSET, HSET, are mentioned under **Notes** section. 

This might be a widely scoped task so please reach out to us on Discord if you have any questions.

### PR DESCRIPTION
**Pull request by pankajjs**
## Issue Number
 - Closes #1324 

## Description
 - This PR adds hyperlink for commands in the documentation for easily navigation.

### CODE REVIEW

**Comment by pankajjs** [comment]
@lucifercr07 @apoorvyadav1111 Could you please review this PR?

File: docs/src/content/docs/commands/BITFIELD_RO.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ BITFIELD_RO key [GET type offset [GET type offset ...]]
 ## Behaviour
 
 - Read-only variant of the BITFIELD command. It is like the original BITFIELD but only accepts GET subcommand.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/BITFIELD_RO.md:28]:
```suggestion
- Read-only variant of the BITFIELD command. It is like the BITFIELD but only accepts GET subcommand.
```

File: docs/src/content/docs/commands/BITFIELD_RO.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ BITFIELD_RO key [GET type offset [GET type offset ...]]
...
-- See original BITFIELD for more details.
+- See original [BITFIELD](/commands/bitfield) for more details.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/BITFIELD_RO.md:29]:
```suggestion
- See [BITFIELD](/commands/bitfield) for more details.
```

File: docs/src/content/docs/commands/COMMAND HELP.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ This command does not accept any parameters.
 
 ## Behavior
 
-The `COMMAND HELP` command outputs help text that lists all the available subcommands for the `COMMAND` command. It is used to understand how each subcommand functions and the available options for those subcommands.
+The `COMMAND HELP` command outputs help text that lists all the available subcommands for the `COMMAND` command. It is used to understand how each subcommand functions and the available options for those subcommands.Please refer to the [COMMAND](/commands/command) command documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/COMMAND HELP.md:29]:
```suggestion
The `COMMAND HELP` command outputs help text that lists all the available subcommands for the [`COMMAND`](/commands/command) command. It is used to understand how each subcommand functions and the available options for those subcommands.
```

File: docs/src/content/docs/commands/DECR.md [hunk 4]
```diff
@@ -101,3 +101,6 @@ OK
 ## Alternatives
 
 You can also use the `DECRBY` command to decrement the value of a key by a specified amount.
+
+## Notes
+- Please refer to the [DECRBY](/commands/decrby) command documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/DECR.md:106]: We can add the information in the Alternatives section and add the hyperlink as follows:
[`DECRBY`](/commands/decrby)

File: docs/src/content/docs/commands/EXPIRE.md [hunk 5]
```diff
@@ -104,3 +104,6 @@ This example shows what happens when trying to set an expiration on a non-existe
 ## Alternatives
 
 - Use `EXPIREAT` command for more precise expiration control based on Unix timestamps
+
+## Notes
+- Please refer to the [TTL](/commands/ttl), [PERSIST](/commands/persist) and [EXPIREAT](/commands/expireat)  commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/EXPIRE.md:109]: same as above. Lets add the infomation in Alternatives section. Also lets keep the style as [`TTL`](...URL..)

File: docs/src/content/docs/commands/EXPIREAT.md [hunk 6]
```diff
@@ -145,3 +145,6 @@ OK
 ## Alternatives
 
 - Use `EXPIRE` command for simpler expiration control based on relative time
+
+## Notes
+- Please refer to the [TTL](/commands/ttl), [PERSIST](/commands/persist) and [EXPIRE](/commands/expire)  commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/EXPIREAT.md:150]: same as above. Lets have this information in Alternatives section.

File: docs/src/content/docs/commands/EXPIRETIME.md [hunk 7]
```diff
@@ -77,3 +77,6 @@ In this example, the key `nonExistentKey` does not exist in the database. The `E
 
 - Use `TTL` to get relative expiration times
 - Use `PTTL` to get relative expiration times in milliseconds
+
+## Notes
+- Please refer to the [TTL](/commands/ttl) and [PTTL](/commands/pttl) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/EXPIRETIME.md:82]: same as above. Lets add the information in Alternatives section.

File: docs/src/content/docs/commands/HEXISTS.md [hunk 8]
```diff
@@ -81,6 +81,7 @@ If the hash `user:1002` does not exist:
 
 ## Notes
 
+- Please refer to the [HGET](/commands/hget) command documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/HEXISTS.md:84]: same as above. Lets add the information in Alternatives section and style as discussed above.

File: docs/src/content/docs/commands/HINCRBY.md [hunk 9]
```diff
@@ -158,3 +158,4 @@ Incrementing the hash value with a very large integer results in an integer over
 ## Notes
 
 - The `HINCRBY` command is a powerful tool for managing counters and numerical values stored in hash fields, making it essential for applications that rely on incremental updates.
+- Please refer to the [HINCRBYFLOAT](/commands/hincrbyfloat) command documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/HINCRBY.md:161]:
```suggestion
- DiceDB also supports [`HINCRBYFLOAT`](/commands/hincrbyfloat) command for incrementing float values. 
```

File: docs/src/content/docs/commands/HINCRBYFLOAT.md [hunk 10]
```diff
@@ -137,3 +137,4 @@ Executing `hincrbyfloat` with a string increment value
 - The `HINCRBYFLOAT` command is a powerful tool for managing floating-point counters and numerical values stored in hash fields, making it essential for applications that require precision in incremental updates.
 - The command operates atomically, meaning it will complete without interruption, making it safe to use in concurrent environments where multiple clients may modify the same hash fields.
 - `HINCRBYFLOAT` can be beneficial in scenarios such as tracking scores in a game, maintaining balances in accounts, or managing quantities in inventory systems where floating-point values are common.
+- Please refer to the [HGET](/commands/hget), [HSET](/commands/hset) and [HINCRBY](/commands/hincrby) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/HINCRBYFLOAT.md:140]: not needed

File: docs/src/content/docs/commands/HKEYS.md [hunk 11]
```diff
@@ -82,7 +82,7 @@ If the hash `user:1003` does not exist:
 - `HGETALL`: The `HGETALL` command retrieves all field-value pairs in a hash as an array, rather than only the field names.
 
 ## Notes
-
+- Please refer to the [HGETALL](/commands/hgetall) command documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/HKEYS.md:85]: same as above. Lets add the information in the Alternatives section and style as discussed above.

File: docs/src/content/docs/commands/HRANDFIELD.md [hunk 12]
```diff
@@ -129,3 +129,4 @@ Passing invalid number of arguments to the `hrandfield` command
 
 - The `HRANDFIELD` command is useful for scenarios where random selection from hash fields is required, such as in games, lotteries, or randomized surveys.
 - The command can return multiple fields at once, allowing for efficient random sampling without the need for multiple calls. This can be particularly advantageous when working with larger hashes.
+- Please refer to the [HEXISTS](/commands/hexists), [HKEYS](/commands/hkeys) and [HSCAN](/commands/hscan) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/HRANDFIELD.md:132]: not needed

File: docs/src/content/docs/commands/HVALS.md [hunk 13]
```diff
@@ -82,7 +82,7 @@ If the hash `user:1003` does not exist:
 - `HGETALL`: The `HGETALL` command retrieves all field-value pairs in a hash, providing both names and values.
 
 ## Notes
-
+- Please refer to the [HGETALL](/commands/hgetall) command documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/HVALS.md:85]: same as above. Lets add the information in Alternatives section and style as discussed above.

File: docs/src/content/docs/commands/INCR.md [hunk 14]
```diff
@@ -95,3 +95,6 @@ Incrementing a key `mykey` with a value that exceeds the maximum integer value:
 
 - You can also use the `INCRBY` command to increment the value of a key by a specified amount.
 - You can also use the `INCRBYFLOAT` command to increment the value of a key by a fractional amount.
+
+## Notes
+- Please refer to the [DECR](/commands/decr), [INCRBY](/commands/incrby) and [INCRBYFLOAT](/commands/incrbyfloat) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/INCR.md:100]:
```suggestion
```
same as above. Lets add the information in Alternatives section and style as discussed above.

File: docs/src/content/docs/commands/LINSERT.md [hunk 15]
```diff
@@ -99,3 +99,6 @@ OK
 - Check Key Type: Before using `LINSERT`, ensure that the key is associated with a list to avoid errors.
 - Handle Non-Existent Keys: Be prepared to handle the case where the key does not exist, as `LINSERT` will return `0` in such scenarios.
 - Use in Conjunction with Other List Commands: The `LINSERT` command is often used alongside other list commands like `RPUSH`, `LPUSH`, `LPOP`, and `RPOP` to manage and process lists effectively.
+
+## Notes
+- Please refer to the [RPUSH](/commands/rpush), [LPUSH](/commands/lpush), [LPOP](/commands/lpop) and [RPOP](/commands/rpop) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/LINSERT.md:104]: same as above. Lets add the information in the Best Practices section and style as discussed above.

File: docs/src/content/docs/commands/LLEN.md [hunk 16]
```diff
@@ -85,3 +85,6 @@ OK
 - Check Key Type: Before using `LLEN`, ensure that the key is associated with a list to avoid errors.
 - Handle Non-Existent Keys: Be prepared to handle the case where the key does not exist, as `LLEN` will return `0` in such scenarios.
 - Use in Conjunction with Other List Commands: The `LLEN` command is often used alongside other list commands like `RPUSH`, `LPUSH`, `LPOP`, and `RPOP` to manage and process lists effectively.
+
+## Notes
+- Please refer to the [RPUSH](/commands/rpush), [LPUSH](/commands/lpush), [LPOP](/commands/lpop) and [RPOP](/commands/rpop) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/LLEN.md:90]: same as above. Lets add the information in the Best Practices section and style as discussed above.

File: docs/src/content/docs/commands/LPOP.md [hunk 17]
```diff
@@ -114,3 +114,6 @@ LPOP mylist secondlist
 - `Use in Conjunction with Other List Commands`: The `LPOP` command is often used alongside other list commands like `RPUSH`, `LPUSH`, `LLEN`, and `RPOP` to manage and process lists effectively.
 
 By understanding and using the `LPOP` command effectively, you can manage list data structures in DiceDB efficiently, implementing queue-like behaviors and more.
+
+## Notes
+- Please refer to the [RPUSH](/commands/rpush), [LPUSH](/commands/lpush), [LLEN](/commands/llen) and [RPOP](/commands/rpop) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/LPOP.md:119]: same as above

File: docs/src/content/docs/commands/LPUSH.md [hunk 18]
```diff
@@ -102,3 +102,6 @@ LPUSH mylist
 - The `LPUSH` command can be used to implement a stack (LIFO) by always pushing new elements to the head of the list.
 
 By understanding the `LPUSH` command, you can efficiently manage lists in DiceDB, ensuring that elements are added to the head of the list as needed.
+
+## Notes
+- Please refer to the [RPUSH](/commands/rpush), [LPOP](/commands/lpop), [LLEN](/commands/llen) and [RPOP](/commands/rpop) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/LPUSH.md:107]: same as above

File: docs/src/content/docs/commands/LRANGE.md [hunk 19]
```diff
@@ -112,3 +112,6 @@ OK
 - Check Key Type: Before using `LRANGE`, ensure that the key is associated with a list to avoid errors.
 - Handle Non-Existent Keys: Be prepared to handle the case where the key does not exist, as `LRANGE` will return an empty array in such scenarios.
 - Use in Conjunction with Other List Commands: The `LRANGE` command is often used alongside other list commands like `RPUSH`, `LPUSH`, `LPOP`, and `RPOP` to manage and process lists effectively.
+
+## Notes
+- Please refer to the [RPUSH](/commands/rpush), [LPUSH](/commands/lpush), [LPOP](/commands/lpop) and [RPOP](/commands/rpop) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/LRANGE.md:117]: same as above

File: docs/src/content/docs/commands/PTTL.md [hunk 20]
```diff
@@ -86,3 +86,6 @@ In this example, the `PTTL` command is used with an extra argument. This results
 ## Alternatives
 
 - `TTL`: Similar to `PTTL` but returns the time-to-live in seconds instead of milliseconds
+
+## Notes
+- Please refer to the [EXPIRE](/commands/expire) and [TTL](/commands/ttl) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/PTTL.md:91]: same as above. Lets add the information in Alternatives section and style as discussed above.

File: docs/src/content/docs/commands/RPOP.md [hunk 21]
```diff
@@ -92,3 +92,6 @@ RPOP mylist secondlist
 - `Use in Conjunction with Other List Commands`: The `RPOP` command is often used alongside other list commands like `RPUSH`, `LPUSH`, `LLEN`, and `LPOP` to manage and process lists effectively.
 
 By understanding the `RPOP` command, you can effectively manage lists in DiceDB, ensuring that you can retrieve and process elements in a LIFO order.
+
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/RPOP.md:95]: Lets add the hyperlinks the existing text.

File: docs/src/content/docs/commands/RPUSH.md [hunk 22]
```diff
@@ -88,3 +88,6 @@ RPUSH mylist
 
 - `Check Key Type`: Before using `RPUSH`, ensure that the key is associated with a list to avoid errors.
 - `Use in Conjunction with Other List Commands`: The `RPUSH` command is often used alongside other list commands like `LLEN`, `LPUSH`, `LPOP`, and `RPOP` to manage and process lists effectively.
+
+## Notes
+- Please refer to the [RPOP](/commands/rpop), [LLEN](/commands/llen), [LPOP](/commands/lpop) and [LPUSH](/commands/lpush) commands documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/RPUSH.md:93]: Lets add the hyperlinks to the existing text.

File: docs/src/content/docs/commands/SMEMBERS.md [hunk 23]
```diff
@@ -90,3 +90,4 @@ OK
 2. The order of returned elements is not guaranteed and may vary between calls
 3. Memory usage scales linearly with the size of the set
 4. This command has O(N) time complexity where N is the set size
+5. Please refer to the [EXISTS](/commands/exists) command documentation for more information
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/SMEMBERS.md:93]:
```suggestion
```

Comment (apoorvyadav1111) [reply docs/src/content/docs/commands/SMEMBERS.md:93]: Lets add the hyperlinks to the existing text.

File: docs/src/content/docs/commands/TTL.md [hunk 24]
```diff
@@ -86,3 +86,6 @@ In this example, the `TTL` command is used with an extra argument. This results
 ## Alternatives
 
 - `PTTL`: Similar to `TTL` but returns the time-to-live in milliseconds instead of seconds
+
+## Notes
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/TTL.md:90]: Lets add the hyperlinks to the existing text.

File: docs/src/content/docs/commands/ZRANK.md [hunk 25]
```diff
@@ -81,3 +81,4 @@ Retrieve both the rank and the score of `member2` in the sorted set `myzset`:
 ## Notes
 
 - This command is particularly useful for implementing leaderboards, pagination in ranked lists, and analytics on data distribution.
+- Please refer to the [ZADD](/commands/zadd) command documentation for more information.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/ZRANK.md:84]: Lets add the hyperlinks to the existing text.

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]
Hi, thanks for raising a PR so quickly, I have left comments, please check. In addition to the comments, I would say not adding any new section but hyperlink the ones in the existing document unless there is no mention of the alternative commands. Additionally lets add customized text in the section than "Please check ... for more information" as this conveys that the link contains more information related to the command page we are at. 

Documentation are essential artifacts for a user to adopt a product and hence the review can sound more like a nit, However, lets add  the changes which conveys the message in the best possible way.
Thanks

**Comment by pankajjs** [comment]
@apoorvyadav1111 Removed new section and added the hyperlink in existing commands. Could you please review it?

**Review by apoorvyadav1111 — APPROVED** [review]
Thank you for this enhancement.

**Pull request merged**